### PR TITLE
[risk=no] DB-339 survey branching logic ui

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Branching the sub questions under the results of main questions.

### CODE REVIEW

File: public-ui/src/app/views/survey-view/survey-view.component.html [hunk 1]
```diff
@@ -168,6 +171,172 @@ <h2 *ngIf="!loading" class="secondary-display">
                         [backgroundColor]="'#ECF1F4'" [domainType]="'surveys'" [chartTitle]="graphToShow"></app-chart>
                     </div>
                   </div>
+                  <div class="row-expansion" *ngIf="a.expanded && a.subQuestions">
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/survey-view/survey-view.component.html]: thanks for doing this. We could probably make this a little cleaner by creating a component.

**Comment by mhigbyflowers** [comment]
Could you send me a screenshot of the UI? I can create a mockup and experiment with color hierarchy.

**Review by mhigbyflowers — APPROVED** [review]

File: public-ui/src/app/views/survey-view/survey-view.component.html (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/survey-view/survey-view.component.html]: Agree working on pulling this apart into component. Will push the changes soon and let you know when its ready for review again.

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/survey-view/survey-view.component.html]: Pulled out graph-menu and chart into a different component. More to de-duplicated will take care in the next prs.

**Pull request merged**